docs/docs/behaviors/leader-key.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Leader Key Behavior
+sidebar_label: Leader Key
+---
+
+## Summary
+
+The leader key behavior when triggered will capture all following key presses and trigger a [Leader Sequence](../features/leader.md) if pressed.
+
+### Behavior Binding
+
+- Reference: `&leader`
+
+Example:
+
+```
+&leader
+```
+
+### Configuration
+
+#### `timeout-ms`
+
+Defines the amount of time to wait to trigger a completed leader sequence. Defaults to 200ms.
+
+To change the timeout term, you can update the existing behavior:
+
+```
+&leader {
+    timeout-ms = <500>;
+};
+
+/ {
+    keymap {
+        ...
+    };
+};
+```
+
+#### `timerless`
+
+By default, the leader key will have a timeout, and will not wait for a sequence to be completed or another key to be pressed. Specify `timerless` if you want a timeout.
```

Comment:
I struggle to understand this part

* Where and how `timerless` should be specified? Is it a possible value for `timeout-ms` or is it a separate option alongside it?
* Why is the opition called "timerless"? The statement says that we should specify it if we want a timeout, it might be a typo and you meant to write "don't want a timeout"? In the latter case, the naming is still a bit non-intuitive, maybe it should be replaced by `no-timeout = <true>`, or even removed in favour of `timeout-ms = <0>`; or `<timeout-ms = <-1>`, or, if timerless *is* a possible value for `timeout-ms`, maybe a shorter `= <false>` or even `= <no>` could be a better option? (shooting up the sky here).

Reply:
- Timerless would be a separate option, defined in the dts for the leader key along with timeout-ms and the other options.
- Yeah, that was a typo, should be "don't want a timeout". Timeout-ms would need to be a positive number for the case of overlaps, unless we had something like overlap-timeout-ms and timeout-ms separate? Timerless is for when no sequence has been completed yet, and once an overlapped sequence is completed, timeout-ms is used.